packages/@glimmer/runtime/lib/compiled/opcodes/debugger.ts

```diff
@@ -23,6 +23,9 @@ function templateOnlyDebugCallback(get: DebugGet): void {
   console.info('Use `get(<@path>)` to debug this template.');
 
   // for example... `get('@foo');`
+  // @ts-ignore
```

Comment:
Does TS have an issue with this, or eslint? In general, I would prefer to not have `@ts-ignore` in the section of code that folks would see when they hit this debugger...

Reply:
`// @ts-expect-error` ? (https://devblogs.microsoft.com/typescript/announcing-typescript-3-9-beta/#ts-expect-error-comments)